### ISSUE TITLE
 Checked radio button is not visible in Edge or Firefox for Windows 10 high contrast mode

### ISSUE THREAD

**Issue by KingMario** [vmware/clarity#4926]
## Describe the accessibility issue

Checked radio button is not visible in Edge or Firefox for Windows 10 high contrast mode.

## WCAG Guideline

N/A

## How to reproduce

1.  Turn on high contrast for Windows 10
2.  Check if a checked radio button of clarity is visible

## Expected behavior

The checked radio button should be visible

## Versions

**App**

- Clarity: [V2 and V3]

**Device:**

- OS: [Windows]
- Browser [Edge and Firefox]

## Additional notes

Override the style of the following in project styles.scss can workaround the issue. Seems the box-shadow style is omitted by Windows high contrast mode.

```
.clr-radio-wrapper input[type="radio"]:checked + label::before {
    border: .25rem solid #0077b8 !important;
    box-shadow: none !important;
}
```

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
